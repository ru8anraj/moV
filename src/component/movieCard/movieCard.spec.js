import React from 'react';
import { shallow } from 'enzyme';
import MovieCard from './movieCard.jsx';

describe('<< MovieCard >>', () => {
  it('should render correctly', () => {
    const component = shallow(<MovieCard />);
  
    expect(component).toMatchSnapshot();
  });
});