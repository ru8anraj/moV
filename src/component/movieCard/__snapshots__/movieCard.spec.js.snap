// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<< MovieCard >> should render correctly 1`] = `
<Card>
  <Image
    as="img"
    ui={false}
    wrapped={true}
  />
  <CardContent>
    <CardHeader
      className="title"
    />
  </CardContent>
  <CardContent
    extra={true}
  >
    <input
      defaultValue="3"
      max="10"
      min="1"
      name="rating"
      step="0.5"
      type="number"
    />
    <Button
      as="button"
      circular={true}
      icon={true}
      size="small"
      style={
        Object {
          "alignSelf": "right",
          "marginLeft": "27px",
        }
      }
    >
      <Icon
        as="i"
        name="checkmark"
        size="small"
      />
    </Button>
  </CardContent>
</Card>
`;
